src/CodeGenerator/AutoloadGenerator.php

```diff
@@ -7,87 +7,93 @@
 
 namespace Zend\Di\CodeGenerator;
 
-use Zend\Code\Generator\DocBlockGenerator;
-use Zend\Code\Generator\FileGenerator;
-use Zend\Code\Generator\ClassGenerator;
-use Zend\Code\Generator\MethodGenerator;
-use Zend\Code\Generator\PropertyGenerator;
-use Zend\Code\Generator\PropertyValueGenerator;
+use SplFileObject;
+use Throwable;
+use Zend\Di\Exception\GenerateCodeException;
+use function array_keys;
```

Comment:
In addition to the previous comment, please add one blank line between import types:

```php
use Zend\Di\Exceptin\GenerateCodeException;

use function array_keys;
```

Reply:
Not sure why github does not mark this, but has been addressed.